### PR TITLE
Add Arm Compiler 6 information and update links

### PR DESCRIPTION
**Pull request by yennster**
This PR updates the offline CLI doc information about toolchains, including updating download links and adding information about ARMC6

### CODE REVIEW

File: docs/tools/offline/cli-setup.md [hunk 1]
```diff
@@ -129,7 +129,8 @@ Mbed CLI requires adding the following to the system `PATH`:
 
 You need to tell Mbed CLI where to find the toolchains that you want to use for compiling. Mbed CLI supports the following toolchains:
 
-- [Arm Compiler 5](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads). Use version 5.06 of Arm Compiler 5. Versions older than 5.06 might be incompatible with the tools.
+- [Arm Compiler 5](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-5). Use version 5.06 of Arm Compiler 5. Versions older than 5.06 might be incompatible with the tools.
+- [Arm Compiler 6](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). Use version 6.8 of Arm Compiler 6. Versions older than 6.8 might be incompatible with the tools.
```
Comment (theotherjimmy) [inline docs/tools/offline/cli-setup.md]: 6.10 please.

File: docs/tools/offline/cli-setup.md [hunk 2]
```diff
@@ -155,7 +156,8 @@ The `-G` switch tells Mbed CLI to set this as a global setting, allowing all pro
...
 
-- `ARM_PATH`: The path to the *base* directory of your Arm Compiler installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARM_PATH`: The path to the *base* directory of your Arm Compiler 5 installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARMC6_PATH`: The path to the *binary* directory of your Arm Compiler 6 installation. This should be the directory containing the binaries for `armar` and friends. For example, if your Arm Compiler 6 executable `armar` is located at `C:/Program Files/ARM/armcc6.8/bin/armar`, you set `ARMC6_PATH` to `C:/Program Files/ARM/armcc6.8/bin`
```
Comment (theotherjimmy) [inline docs/tools/offline/cli-setup.md]: Technically, the tools look for `armclang`, but they're always in the same folder.

File: docs/tools/offline/cli-setup.md (hunk 1 above)
Comment (yennster) [reply docs/tools/offline/cli-setup.md]: @theotherjimmy Ok, I took that 6.8 number from the Mbed CLI readme, so that will need to get changed too

**Comment by AnotherButler** [comment]
@theotherjimmy Do these changes address your concerns?

File: docs/tools/offline/cli-setup.md [hunk 3]
```diff
@@ -155,7 +156,8 @@ The `-G` switch tells Mbed CLI to set this as a global setting, allowing all pro
...
 
-- `ARM_PATH`: The path to the *base* directory of your Arm Compiler installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARM_PATH`: The path to the *base* directory of your Arm Compiler 5 installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARMC6_PATH`: The path to the *binary* directory of your Arm Compiler 6 installation. This should be the directory containing the binaries for `armclang` and friends. For example, if your Arm Compiler 6 executable `armclang` is located at `C:/Program Files/ARM/armcc6.8/bin/armclang`, you set `ARMC6_PATH` to `C:/Program Files/ARM/armcc6.8/bin`
```
Comment (theotherjimmy) [inline docs/tools/offline/cli-setup.md]: This also references 6.8. Could you update the paths to 6.10?

**Comment by yennster** [comment]
@theotherjimmy Done

**Pull request merged**